frontend/public/index.html

```diff
@@ -24,7 +24,10 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+
+    <script defer src="https://use.fontawesome.com/releases/v5.3.1/js/all.js"></script>
```

Comment:
I would rather not load a script from an external source. can you please investigate this library? it seems to be the official way to integrate font awesome to a react project: https://github.com/FortAwesome/react-fontawesome

from `frontend` directory run:
```
$ yarn add @fortawesome/fontawesome-svg-core
$ yarn add @fortawesome/free-solid-svg-icons
$ yarn add @fortawesome/react-fontawesome
```

here's an example usage:
```
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
import { faCoffee } from '@fortawesome/free-solid-svg-icons'

const element = <FontAwesomeIcon icon={faCoffee} />
```

let me know if you encounter any issues with this

Reply:
you may also directly specify the icon name as a string:
```
<FontAwesomeIcon icon="coffee" />
```